src/doc_table.c

```diff
@@ -208,14 +211,27 @@ t_docId DocTable_Put(DocTable *t, const char *s, size_t n, double score, u_char
 
   sds keyPtr = sdsnewlen(s, n);
 
-  RSDocumentMetadata *dmd = rm_calloc(1, sizeof(RSDocumentMetadata));
-  dmd->keyPtr = keyPtr;
-  dmd->score = score;
-  dmd->flags = flags;
-  dmd->payload = dpl;
-  dmd->maxFreq = 1;
-  dmd->id = docId;
-  dmd->sortVector = NULL;
+  RSDocumentMetadata *dmd = NULL;
+  if (t->tempDmdDict) {
```

Comment:
This is no longer needed, it was added before we decided that each index must have a rule.

Reply:
We will not save the doctable in the aux fields